src/SFA.DAS.ToolsNotifications.Api/Controllers/NotificationController.cs

```diff
@@ -32,14 +32,9 @@ public async Task<ActionResult<Notification>> Get()
         }
 
         [HttpPost]
-        public async Task Post(string title, string description)
+        public async Task Post([FromBody] Notification notification)
```

Comment:
I'd probably introduce a DTO here if you want to use it on the API endpoint so it's not affected by any upstream changes